### PR TITLE
Added Swedish translation

### PR DESCRIPTION
**Pull request by H2SO4JB**
Hi,
This is my first "Fork repo" and "Pull request" ever, so I hope I've done all right.
Otherwise, please tell me what I've done wrong.  :-)

### CODE REVIEW

File: readme/README.es_ES.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Herramienta de traducción & menú traducido para usuarios finales de Sublime Te
 - es_ES [Español](README.es_ES.md)
 - hy [Հայերեն](README.hy.md)
 - ru [Русский](README.ru.md)
+- sv_SE [Svenska](readme/README.sv_SE.md)
```
Comment (zam1024t) [inline readme/README.es_ES.md]: Should be `sv_SE [Svenska](README.sv_SE.md)`

File: readme/README.es_ES.md [hunk 2]
```diff
@@ -80,9 +81,10 @@ Herramienta de traducción & menú traducido para usuarios finales de Sublime Te
 - Enviar pull request
 
 # Idiomas & Contribuyentes
-- es_ES Español by [Dastillero](https://github.com/dap39)*
+- es_ES Español *by [Dastillero](https://github.com/dap39)*
```
Comment (zam1024t) [inline readme/README.es_ES.md:84]: Nice! You fix it.

File: readme/README.hy.md [hunk 3]
```diff
@@ -12,6 +12,7 @@
 - es_ES [Español](README.es_ES.md)
 - hy [Հայերեն](README.hy.md)
 - ru [Русский](README.ru.md)
+- sv_SE [Svenska](readme/README.sv_SE.md)
```
Comment (zam1024t) [inline readme/README.hy.md]: As above.

File: readme/README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+- sv_SE Svenska *by [H2SO4JB](https://github.com/H2SO4JB)*
+- zh_CN 简体中文 *by [Zam](https://github.com/zam1024t)*
+- zh_TW 繁体中文 *by [Zam](https://github.com/zam1024t)*
+
+# Related discuss
+- https://github.com/wbond/package_control_channel/pull/5665
+- https://github.com/rexdf/ChineseLocalization/issues/10
+
+# License
+[The MIT License](LICENSE)
```
Comment (zam1024t) [inline readme/README.md:96]: This file should move to the parent dir.

**Comment by H2SO4JB** [comment]
I think I've fixed everything now.
But I don't know how to delete the README.md file in the /readme folder.

**Comment by zam1024t** [comment]
You did a good job.
We can release the new version with Swedish translation after correct the link in **readme/*.md**.

**Comment by H2SO4JB** [comment]
Okay, thanks!
Have I understood correctly that everything is done now?

**Pull request merged**